Assets/MixedRealityToolkit.SDK/Experimental/HandCoach/Scripts/InteractionHint.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.Collections;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    public class InteractionHint : MonoBehaviour
+    {
+        private GameObject VisualsRoot;
+
+        [Tooltip("React to hand tracking state to hide visuals when hands are being tracked.  If false, only the customShouldHideHands function will be evaluated.")]
+        public bool checkHandTracking = true;
```

Comment:
Please make these private, serialized with property accessors to be consistent with the rest of mrtk. See guideline [Access Modifiers](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers
)

Reply:
In the guidelines, the example shows this:
public int MyVariable { get; protected set; } = 0;

Should I follow that pattern, or do the private field with the public version referencing that?

Reply:
Answered my own question. The example way doesn't expose those fields in the inspector